exporter/collector/googlemanagedprometheus/monitoredresource_test.go

```diff
@@ -320,10 +320,10 @@ func TestMapToPrometheusTarget(t *testing.T) {
 		{
 			desc: "Attributes from GCE with environment label",
 			resourceLabels: map[string]string{
-				"cloud.platform":      "gcp_compute_engine",
-				"cloud.region":        "us-central1",
-				"service.name":        "service-name",
-				"service.instance.id": "1234759430923053489543203",
```

Comment:
should we add a test for if `service.instance.id` and `host.id` are both set? Or is that not possible?

Reply:
It is possible.  I can test for that

Reply:
added